src/application/deployer.go

```diff
@@ -18,17 +18,16 @@ func StartDeploy(deployConfig types.DeployConfig) error {
 		return err
 	}
 
-	deployConfig.LogChannel <- "Setting up AWS account..."
+	deployConfig.Logger.Log("Setting up AWS account...")
 	err = aws.InitAccount(deployConfig.AwsConfig)
 	if err != nil {
 		return err
 	}
 
-	deployConfig.LogChannel <- fmt.Sprintf("Building %s %s...\n\n", deployConfig.AppConfig.Name, deployConfig.AppConfig.Version)
+	deployConfig.Logger.Log(fmt.Sprintf("Building %s %s...\n\n", deployConfig.AppConfig.Name, deployConfig.AppConfig.Version))
```

Comment:
Can be a separate PR but would be cool if we provided a `Logf` method like

```go
deployConfig.Logger.Logf("Building %s %s...\n\n", deployConfig.AppConfig.Name, deployConfig.AppConfig.Version)
```

Reply:
Will do separately